pubsub/src/main/java/com/github/twitch4j/pubsub/TwitchPubSubBuilder.java

```diff
@@ -61,6 +71,12 @@ public class TwitchPubSubBuilder {
     @With
     private int wsPingPeriod = 15_000;
 
+    /**
+     * WebSocket Connection Backoff Strategy
+     */
+    @With
+    private IBackoffStrategy connectionBackoffStrategy = null;
```

Comment:
This technically changes the default backoff settings for pubsub from [master](https://github.com/twitch4j/twitch4j/blob/af5744a5a4e142f443cb8ccca1d54163d60c5079/chat/src/main/java/com/github/twitch4j/chat/TwitchChat.java#L250-L256), but is relatively minor so this is fine 👍